apis/certificatemanager/v1alpha1/certificatemanagerdnsauthorization_types.go

```diff
@@ -0,0 +1,131 @@
+	// +optional
+	Name *string `json:"name,omitempty"`
+
+	/* Type of the DNS Resource Record. */
+	// +optional
+	Type *string `json:"type,omitempty"`
+}
+
+// CertificateManagerDNSAuthorizationStatus defines the config connector machine state of CertificateManagerDNSAuthorization
+type CertificateManagerDNSAuthorizationStatus struct {
+	/* Conditions represent the latest available observations of the
+	   CertificateManagerDNSAuthorization's current state. */
+	Conditions []v1alpha1.Condition `json:"conditions,omitempty"`
+	/* The structure describing the DNS Resource Record that needs to be added
+	to DNS configuration for the authorization to be usable by
+	certificate. */
+	// +optional
+	DnsResourceRecord []DnsauthorizationDnsResourceRecordStatus `json:"dnsResourceRecord,omitempty"`
+
+	/* ObservedGeneration is the generation of the resource that was most recently observed by the Config Connector controller. If this is equal to metadata.generation, then that means that the current reported status reflects the most recent desired state of the resource. */
+	// +optional
+	ObservedGeneration *int64 `json:"observedGeneration,omitempty"`
+
+	/* NOTYET
+	// A unique specifier for the CertificateManagerDNSAuthorization resource in GCP.
+	// +optional
+	ExternalRef *string `json:"externalRef,omitempty"`
+	*/
+
+	/* NOTYET
+	// ObservedState is the state of the resource as most recently observed in GCP.
+	// +optional
+	ObservedState *CertificateManagerDNSAuthorizationObservedState `json:"observedState,omitempty"`
+	*/
+}
+
+/* NOTYET
+// CertificateManagerDNSAuthorizationSpec defines the desired state of CertificateManagerDNSAuthorization
+// +kcc:proto=google.cloud.certificatemanager.v1.DnsAuthorization
+type CertificateManagerDNSAuthorizationObservedState struct {
+}
+*/
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:categories=gcp,shortName=gcpcertificatemanagerdnsauthorization;gcpcertificatemanagerdnsauthorizations
+// +kubebuilder:subresource:status
+// +kubebuilder:metadata:labels="cnrm.cloud.google.com/managed-by-kcc=true";"cnrm.cloud.google.com/system=true"
+// +kubebuilder:printcolumn:name="Age",JSONPath=".metadata.creationTimestamp",type="date"
+// +kubebuilder:printcolumn:name="Ready",JSONPath=".status.conditions[?(@.type=='Ready')].status",type="string",description="When 'True', the most recent reconcile of the resource succeeded"
+// +kubebuilder:printcolumn:name="Status",JSONPath=".status.conditions[?(@.type=='Ready')].reason",type="string",description="The reason for the value in 'Ready'"
+// +kubebuilder:printcolumn:name="Status Age",JSONPath=".status.conditions[?(@.type=='Ready')].lastTransitionTime",type="date",description="The last transition time for the value in 'Status'"
+// +kubebuilder:metadata:labels="cnrm.cloud.google.com/managed-by-kcc=true";"cnrm.cloud.google.com/stability-level=alpha";"cnrm.cloud.google.com/tf2crd=true"
```

Comment:
The cnrm.cloud.google.com/stability-level needs to be removed to fix the presubmit failure.